### PR TITLE
changed binder's link to reflect the new name of master branches from…

### PR DESCRIPTION
**Pull request by omar-3**

### CODE REVIEW

**Comment by omar-3** [comment]
Binder's build logs still have this error that i can't resolve

````
Pip subprocess error:
  Running command git clone -q https://github.com/pysal/spaghetti.git /tmp/pip-req-build-1koxr8zs
  WARNING: Did not find branch or tag 'master', assuming revision or ref.
  Running command git checkout -q master
  error: pathspec 'master' did not match any file(s) known to git.
ERROR: Command errored out with exit status 1: git checkout -q master Check the logs for full command output.


CondaEnvException: Pip failed
````

**Comment by jGaboardi** [comment]
@omar-3 Thanks for the PR, these are nice catches that I overlooked.

> Binder's build logs still have this error that i can't resolve.

Regarding this error, I think it may resolve itself after merging in this PR. My guess is that the `binder` is trying to build from the current `main` branch, but the `environment.yaml` in `main` still lists `git+https://github.com/pysal/spaghetti.git@master`. I'll go ahead and merge this and see if that does the trick.

**Review by jGaboardi — APPROVED** [review]

**Comment by jGaboardi** [comment]
Merging this fixed the issue! Thanks again for the PR.

**Pull request merged**
